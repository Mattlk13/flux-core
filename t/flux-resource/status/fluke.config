{
  "queues": {
    "batch": {
      "requires": [
        "batch"
      ]
    },
    "debug": {
      "requires": [
        "debug"
      ]
    }
  }
}
