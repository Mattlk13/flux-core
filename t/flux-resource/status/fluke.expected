     STATE NNODES RANKS           NODELIST
     avail     85 3-13,16-20,22-39,41-57,59-61,63-72,75,77-88,90-93,96-99 fluke[6-16,19-23,25-42,44-60,62-64,66-75,78,80-91,93-96,99-102]
    avail*     11 14,21,58,62,73-74,76,89,94-95,100 fluke[17,24,61,65,76-77,79,92,97-98,103]
   exclude      3 0-2             fluke[1,3,108]
   drained      2 15,40           fluke[18,43]
